spec/controllers/api/steps/steps_destroy_spec.rb

```diff
@@ -0,0 +1,33 @@
+require 'spec_helper'
+
+describe Api::StepsController do
+
+  include Devise::TestHelpers
+
+  describe '#create' do
+    let(:sequence) { FactoryGirl.create(:sequence) }
+    let(:step) { sequence.steps[0] }
+    let(:user) { sequence.user }
+
+    it 'destroys a step sequence' do
+      sign_in user
+      input = { sequence_id: sequence._id.to_s,
+                id: step._id.to_s }
+      before = sequence.steps.count
+      delete :destroy, input
+      expect(response.status).to eq(200)
+      expect(sequence.reload.steps.count).to be < before
+    end
+
+    it 'handles 404 for step' do
+      sign_in user
+      input = { sequence_id: sequence._id.to_s,
+                id: '0000000000' }
+      before = sequence.steps.count
```

Comment:
Weird. My file renaming seems to have confused hound.

Either way, it's been fixed. see `steps/destroy_spec.rb`